tools/esdlcmd/esdl-publish.cpp

```diff
@@ -160,11 +160,11 @@ class EsdlPublishCmd : public EsdlPublishCmdCommon
     void usage()
     {
         printf("\nUsage:\n\n"
-                "esdl publish <servicename> <filename.ecm> [command options]\n\n"
-                "   <servicename>        The ESDL defined ESP service to publish\n"
-                "   <filename.ecm>       The ESDL file containing service definition\n"
+                "esdl publish <servicename> <filename.(ecm|esdl)> [command options]\n\n"
```

Comment:
this particular command actually accepts esdl or esxdl, we could open the file and inspect the content to determine if the file contains esdl which needs to be processed to esxdl first, or if it's xml which it attempts to load regardless of extension. Right now we use the ecm/esdl extension to determine if the content needs to be processed (the usage needs to reflect this)